ci/tests/puppeteer/cas.js

```diff
@@ -327,6 +364,22 @@ exports.launchSamlSp = async (idpMetadataPath, samlSpDir, samlOpts = []) => {
     return exec;
 }
 
+exports.stopActuator = async (baseUrl) => {
```

Comment:
You should rename this to "shutDownCas" or something to tat degree. The name here makes it sounds like you're shutting down or stopping the actuator.